packages/docusaurus-utils/src/markdownParser.ts

```diff
@@ -36,17 +36,17 @@ export function createExcerpt(fileString: string): string | undefined {
       // Remove HTML tags.
       .replace(/<[^>]*>/g, '')
       // Remove Title headers
-      .replace(/^\#\s*([^#]*)\s*\#?/gm, '')
+      .replace(/^#\s*([^#]*)\s*#?/gm, '')
```

Comment:
I'm not totally confident with the changes here, but as far as I know, regex is wise enough to know when to treat characters as special ones and when to treat them as plain text. Therefore I just followed the ESLint recommendations.

Reply:
agree